src/config/field-rules.js

```diff
@@ -796,4 +796,17 @@ export default new Map([
             }],
         },
     ]],
+    ['dataSet', [
+        {
+            field: 'openPeriodsAfterCoEndDate',
```

Comment:
I wouldn't abbreviate category options, I didn't know what it actually meant until I saw the rendered field

Reply:
has to match the schema